react/src/utils/backendrouter.js

```diff
@@ -1,16 +1,17 @@
 const DEFAULT_BACKEND = "flask"
+
 const SUPPORTED_BACKEND_TYPES = {
   "flask": {
       "test": "http://localhost:8080",
       "production": process.env.FLASK_BACKEND || process.env.REACT_APP_BACKEND
   },
   "express": {
       "test": "http://localhost:8088",
-      "production": process.env.EXPRESS_BACKEND
+      "production": process.env.REACT_APP_EXPRESS_BACKEND
```

Comment:
This file looks good. Note that we should update the `react/.env.example` to have `REACT_APP_` prefixes

https://github.com/sentry-demos/application-monitoring/blob/master/react/.env.example

i.e.

```
EXPRESS_BACKEND=
SPRINGBOOT_BACKEND=
FLASK_BACKEND
```

should all be updated to have `REACT_APP_` prefix.

Reply:
good call